test/controllers/config_reports_controller_test.rb

```diff
@@ -30,18 +30,18 @@ def test_index
     host = FactoryBot.create(:host)
     FactoryBot.create(:config_report, :host => host)
 
-    # "any context"
-    get :index, {format: :csv}, set_session_user
+    get :index, params: {format: :csv}, session: set_session_user.merge(location_id: FactoryBot.create(:location).id)
     assert_response :success
-    assert_equal 2, response.body.lines.size
+    assert_equal 1, response.body.lines.size
 
-    get :index, {format: :csv}, set_session_user.merge(location_id: host.location_id)
+    get :index, {format: :csv}, session: set_session_user.merge(location_id: host.location_id)
     assert_response :success
     assert_equal 2, response.body.lines.size
 
-    get :index, {format: :csv}, set_session_user.merge(location_id: FactoryBot.create(:location).id)
+    # "any context"
```

Comment:
is there a reasoning behind the reordering?

Reply:
Yes, the tests pass now, fail if they are ordered the other way

Reply:
@xprazak2 any idea why?

Reply:
No, but I can look into it